nixos/modules/services/networking/iperf.nix

```diff
@@ -0,0 +1,92 @@
+{ config, lib, pkgs, ... }: with lib;
+let
+  cfg = config.services.iperf3;
+
+  api = {
+    enable = mkEnableOption "iperf3 network throughput testing server";
+    port = mkOption {
+      type        = types.int;
+      default     = 5201;
+      description = "Server port to listen on for iperf3 client requsts";
+    };
+    affinity = mkOption {
+      type        = types.nullOr types.string;
+      default     = null;
+      description = "CPU affinity for the process";
+    };
+    bind = mkOption {
+      type        = types.nullOr types.string;
+      default     = null;
+      description = "Bind to the specific interface associated with the given address";
+    };
+    verbose = mkOption {
+      type        = types.bool;
+      default     = false;
+      description = "Give more detailed output";
+    };
+    forceFlush = mkOption {
+      type        = types.bool;
+      default     = false;
+      description = "Force flushing output at every interval";
+    };
+    debug = mkOption {
+      type        = types.bool;
+      default     = false;
+      description = "Emit debugging output";
+    };
+    rsaPrivateKey = mkOption {
+      type        = types.nullOr types.string;
```

Comment:
You can make the type `types.path` here, which forces it to be an absolute one (starting with "/"). Same for `authorizedUsersFile`